packages/react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -137,15 +137,16 @@ const Checkbox = React.forwardRef(
 
     const checkboxGroupInstanceId = useId();
 
-    const helperId = !helperText
+    const hasHelper = helperText !== null && helperText !== undefined;
```

Comment:
This suggestion applies here and elsewhere:

```suggestion
    const hasHelper = helperText !== null && typeof helperText !== 'undefined';
```

Reply:
Sorry for the delay, comments addressed.